common/include/Utilities/IniInterface.h

```diff
@@ -70,10 +70,11 @@ class IniInterface
 	void EnumEntry( const wxString& var, T& value, const wxChar* const* enumArray=NULL, const T defvalue=(T)0 )
 	{
 		int tstore = (int)value;
+		int defaultvalue = (int)defvalue;
```

Comment:
I'm not sure I fully understand your idea, so basically you want to replace the static casts with a template function call? ``as_int(enum)`` instead of ``static_cast<uint8>(enum)`` to directly cast to the underlying type of the enum?